pkg/core/pipeline/conditions.go

```diff
@@ -42,22 +42,24 @@ func (p *Pipeline) RunConditions() (globalResult bool, err error) {
 				p.Sources[condition.Config.SourceID].Output +
 				p.Sources[condition.Config.SourceID].Config.Postfix)
 
+		if err != nil {
+			// Show error to end user
+			logrus.Error(err)
+			if condition.Result != result.SUCCESS {
```

Comment:
Does this check has to be done here instead of l51 outside the "if err"?

Reply:
see updated code